tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -385,9 +412,20 @@ <h3>No profile data was found.</h3>
         value: "ip-address",
       },
       _profileDuration: {
-        type: Number,
+        type: String,
         value: 1000,
       },
+      _profileNumRetry: {
+        type: String,
+        value: 3,
```

Comment:
If `type` is `String`, then `value` should be `"3"`, not `3`, right?

(Likewise for `_profileDuration` above.)

Reply:
Yep, done, thanks :)